test/IntelFPGAMemoryAttributesForStruct.ll

```diff
@@ -556,6 +583,38 @@ entry:
   ret void
 }
 
+; Function Attrs: norecurse nounwind
+define spir_func void @_Z27field_force_pow2_depth_attrv() #3 {
+entry:
+  %s = alloca %struct.force_pow2_depth_st, align 4
+  %0 = bitcast %struct.force_pow2_depth_st* %s to i8*
+  call void @llvm.lifetime.start.p0i8(i64 4, i8* %0) #5
+  ; CHECK-LLVM: %[[FLD_FP2_SCT:.*]] = getelementptr inbounds %struct.force_pow2_depth_st, %struct.force_pow2_depth_st* %{{[a-zA-Z0-9]+}}, i32 0, i32 0
```

Comment:
IMO, the mapping on a particular global (line 280-281) had best be checked. The globals could be pre-extracted by `CHECK-DAG`s, I believe.